presto-memory-context/src/main/java/com/facebook/presto/memory/context/LocalMemoryContext.java

```diff
@@ -28,9 +28,28 @@
 
     long getBytes();
 
+    /**
+     * When this method returns the bytes tracked by this LocalMemoryContext will be updated immediately.
```

Comment:
nit: place a comma after `When this method returns` 

`s/will be updated immediately./has been updated.`